tests/data/make_central_repo.sh

```diff
@@ -19,4 +22,4 @@ find central_repo/DECam/calib/20150313T000000Z -name "*.fits" -execdir sh -c '>
 find central_repo/DECam/calib/curated -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
 # find central_repo/skymaps -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
 
-sqlite3 central_repo/gen3.sqlite3 "update file_datastore_records set file_size=0;"
+sqlite3 central_repo/gen3.sqlite3 'update file_datastore_records set file_size=0 where path != "DECam/calib/unbounded/camera/camera_DECam_DECam_calib_unbounded.fits" and path != "skymaps/skyMap/skyMap_deepCoadd_skyMap_skymaps.pickle";'
```

Comment:
I'm very puzzled how it works in the main branch.  None of the "file_size" in the main branch's gen3.sqlite3 is 0. So I suspect that this update wasn't run last time. 
If I make all file_size 0 (i.e. run the original sqlite3 command), `tests/test_middleware_interface.py` would fail with Datastore Integrity failure from size mismatch.

Reply:
Unless... butler datastore is not bothered by the size mismatch for those fits files, because those never are read anyway.   Looks like the checking is when the file is read into memory [(error here)](https://github.com/lsst/daf_butler/blob/5c8060ee775b232e4984be3f3f03434338682e2c/python/lsst/daf/butler/datastores/fileDatastore.py#L1244 ).  So, only the sizes of camera and skymap matter, which are the two files that aren't and can't be emptied. 

So, my guess is that this sqlite3 update was just not run last time?

Reply:
I'll add a comment to the file.